mail/backup_mailer/src/opnsense/mvc/app/library/OPNsense/Backup/Mailer.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace OPNsense\Backup;
+
+require_once('phpmailer/PHPMailerAutoload.php');
+
+use OPNsense\Core\Config;
+use OPNsense\Backup\MailerSettings;
+
+/**
+ * Class mail backup
+ * @package OPNsense\Backup
+ */
+class Mailer extends Base implements IBackupProvider
+{
+
+    /**
+     * get required (user interface) fields for backup connector
+     * @return array configuration fields, types and description
+     */
+    public function getConfigurationFields()
+    {
+        $fields = array();
+
+        $fields[] = array(
+            "name" => "MailEnabled",
```

Comment:
I would recommend you to call it only Enabled. Mail is clear due to the model and its mountpoint.